docs/02.API-REFERENCE.md

```diff
@@ -1495,7 +1497,7 @@ someplace_in_the_code (void)
 
 Registers an external magic string array.
 
-*Notes*:
+*Note*:
```

Comment:
My two cents: plural (i.e., "Notes") is quite OK when there are multiple comments.

(Of course, it depends whether "Note" is a verb or a noun here. If it stands for "Note that...", then the trailing "s" is incorrect. If it stands for the noun "Note", then my above comment holds.)

Reply:
I think it is very easy to forget updateing note / notes after a comment is added / removed. I would prefer to use a single form.
Alternative: `Additional info:` It has no plural problem.